README.md

```diff
@@ -147,6 +147,35 @@ in `rustfmt.toml`.
 [Rust](https://github.com/rust-lang/rust) code.
 - Alternatively, run `make format-lint` to run both at the same time.
 
+# Run Benchmarking
+
+In order to achieve a certain block time, for example 6 seconds per block,
+there is a limited number of lines of code can be run during that time frame.
+When writing a pallet function, the developer is responsible for calculating its
+computational complexity, which is called **weight**. The process of
+determining that complexity, or simply put, time cost is called benchmarking.
+
+1. Build your Frequency node binary with the `runtime-benchmarks` feature flag:
```

Comment:
Do we want to wait on this PR until #280 is done? That would change this a fair amount to how to run them on a PR.

Running them locally would then be the section you have here, but wouldn't really be used except locally.